configurations/operator/perf-regression-latency-multitenant-2-clients.yaml

```diff
@@ -0,0 +1,27 @@
+test_duration: 540
+user_prefix: 'perf-regression-latency-2-clients'
+
+# NOTE: deploy 8 K8S nodes of the 'loader' type and then create 4 loader pods per DB cluster
+n_loaders: 8
+k8s_n_loader_pods_per_cluster: 4
+
+prepare_write_cmd: [
+    "cassandra-stress write no-warmup cl=ALL n=15000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=20 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..15000000",
+    "cassandra-stress write no-warmup cl=ALL n=15000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=20 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=15000001..30000000",
+    "cassandra-stress write no-warmup cl=ALL n=15000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=20 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=30000001..45000000",
+    "cassandra-stress write no-warmup cl=ALL n=15000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=20 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=45000001..60000000",
+]
+# NOTE: 'stress_cmd_w', 'stress_cmd_r' and 'stress_cmd_m' are lists of strs where
+#       one str is dedicated for one of DB clusters.
+#       If it is just one str then the same command will be used for each of DB clusters
+
+# NOTE: main values divided by 2
+# read = 10000 / 2
+stress_cmd_r: [
+    "cassandra-stress read no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate 'threads=100 throttle=5000/s' -col 'size=FIXED(1024) n=FIXED(1)' -pop 'dist=gauss(1..60000000,30000000,15000000)' ",
+    "cassandra-stress read no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate 'threads=100 throttle=5000/s' -col 'size=FIXED(1024) n=FIXED(1)' -pop 'dist=gauss(1..60000000,30000000,15000000)' ",
+]
+# write = 15000 / 2
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate 'threads=100 throttle=7500/s' -col 'size=FIXED(1024) n=FIXED(1)' -pop 'dist=gauss(1..60000000,30000000,15000000)' "
+# mixed = 9000 / 2
+stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate 'threads=100 throttle=4500/s' -col 'size=FIXED(1024) n=FIXED(1)' -pop 'dist=gauss(1..60000000,30000000,15000000)' "
```

Comment:
why for stress_cmd_r we split into 2 and the rest aren't?

Reply:
Number of elements for one of these shows the number of clusters. All other will be reused for each of the clusters with the same values.

Reply:
ok, it's not trivial.
I would do it as a parameter of how many tenants and even the the throttle I would set as parameter.
But we can leave it as is for now.